tensorrt_llm/_torch/models/modeling_nemotron_h.py

```
@@ -528,6 +528,14 @@ def load_weights(self, weights: Dict, weight_mapper: BaseWeightMapper):
         new_weights = weight_mapper.preprocess_weights(weights)
         super().load_weights(weights=new_weights, weight_mapper=weight_mapper)
 
+    @classmethod
+    def get_model_defaults(cls, llm_args: 'BaseLlmArgs') -> dict:
```

Ruff (0.14.14)

[warning] Unused class method argument: `llm_args`

(ARG003)